storage/Utils/Lock.cc

```diff
@@ -33,7 +33,7 @@
 #include "storage/Utils/Lock.h"
 
 
-#define LOCKDIR "/run/libstorage"
+#define LOCKDIR "/run/libstorage-ng"
```

Comment:
I don't think that having different lock files for the libraries is a good idea.